zipline/testing/fixtures.py

```diff
@@ -598,6 +598,8 @@ class WithSimParams(WithTradingEnvironment):
     SIM_PARAMS_START = alias('START_DATE')
     SIM_PARAMS_END = alias('END_DATE')
 
+    SIM_PARAMS_CALENDAR = get_calendar('NYSE')
```

Comment:
I noticed that we were only using `cls.trading_calendar` for testing; figured I'd switch it to just use `NYSE`. But now that you've pointed it out, we'd probably not want this change so we can test across whatever calendar is passed in

Reply:
Ah I also saw test failures because the default `trading_calendar` is None:

e.g.
```
======================================================================
FAIL: test_schedule_function_time_rule_positionally_misplaced (tests.test_algorithm.TestAlgoScript)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/freddiev4/Documents/quantopian/zipline/tests/test_algorithm.py", line 2272, in test_schedule_function_time_rule_positionally_misplaced
    data_frequency='minute'
  File "/Users/freddiev4/Documents/quantopian/zipline/zipline/utils/factory.py", line 62, in create_simulation_parameters
    trading_calendar=trading_calendar,
  File "/Users/freddiev4/Documents/quantopian/zipline/zipline/finance/trading.py", line 141, in __init__
    "Must pass in trading calendar!"
AssertionError: Must pass in trading calendar!
```

Reply:
I'm not sure that failure is related, since that test is calling `create_simulation_parameters` directly, not using the `self.sim_params` attribute provided by the fixture